schedule.qmd

```diff
@@ -0,0 +1,35 @@
+---
+title: schedule
+---
+
+🗓 July 25 and 26, 2022\
+⏰ 9:00 - 17:00\
+🏨 \[ADD ROOM\]\
+✍️ [rstd.io/conf](http://rstd.io/conf)
+
+------------------------------------------------------------------------
+# Day 1
+
+| Time          | Activity                                                                                 |
+|:----------------------------------|:------------------------------------|
+| 09:00 - 10:30 | [Session 1: Introduction and the whole game](materials/day-1-session-1-introduction.qmd) |
+| 10:30 - 11:00 | *Coffee break*                                                                           |
+| 11:00 - 12:30 | [Session 2: Documentation - Minimal](materials/day-1-session-2-doc-minimal.qmd)          |
+| 12:30 - 13:30 | *Lunch break*                                                                            |
+| 13:30 - 15:00 | [Session 3: Unit testing](materials/day-1-session-3-testing.qmd)                         |
+| 15:00 - 15:30 | *Coffee break*                                                                           |
+| 15:30 - 17:00 | [Session 4: Documentation - Sharing]()                                                   |
```

Comment:
`quarto build` scans for `.qmd` links, then renders those automatically (yay!), but if the `.qmd` file is not there, the build fails - so I removed this link for now.